cgmes/cgmes-extensions/src/main/java/com/powsybl/cgmes/extensions/CgmesControlAreas.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.cgmes.extensions;
+
+import java.util.Collection;
+
+import com.powsybl.commons.extensions.Extension;
+import com.powsybl.iidm.network.Network;
+
+/**
+ * @author Marcos de Miguel <demiguelm at aia.es>
+ */
+public interface CgmesControlAreas extends Extension<Network> {
```

Comment:
You need this second extension because you can find several `CgmesControlArea` in the same network?